rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: Tuple[str, str, str],
+            contract: EthereumContract,
+    ) -> List:
+        """Acts a fallback for `get_basic_contract_info` when a RemoteError happens.
+        'decimals' property defaults to 18.
```

Comment:
Having the custom type would help here.

```suggestion
        """Acts a fallback for `get_basic_contract_info` when information about the token can't be retrieved.
        - 'decimals' property defaults to 18.
        - `name` and `symbol` default to None
```